state/executor.go

```diff
@@ -1044,6 +1044,11 @@ func (t *Transition) SetCodeDirectly(addr types.Address, code []byte) error {
 	return nil
 }
 
+// SetNoBaseFee sets no base fee flag to the transition runtime context
+func (t *Transition) SetNoBaseFee(noBaseFee bool) {
```

Comment:
Consider to rename accordingly, in case field name gets changed.

Reply:
Done